tasks/whitelist-env.yml

```diff
@@ -0,0 +1,25 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: eu.gcr.io/census-gcr-rm/rm/census-rm-toolbox
+params:
+  GCP_PROJECT_NAME:
+  ADMIN_SERVICE_ACCOUNT_JSON:
```

Comment:
Isn't there a mismatch in the param name - `SERVICE_ACCOUNT_JSON` v `ADMIN_SERVICE_ACCOUNT_JSON`?